docs/source/changelog.rst

```diff
@@ -13,6 +13,19 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
+v0.15.0 / 2020-06-26
+--------------------
+
+New Features
+++++++++++++
+
+Enhancements
+++++++++++++
+- (:pr:`239`) OpenMM - extracts CMILES from ``AtomicInput`` if possible for consistent atom ordering
+
+Bug Fixes
++++++++++
+
```

Comment:
feel free to start the v0.17 changelog, but this looks like a forgotten start.

Reply:
Yeah sorry, this came from a polluted master branch on my local repo. Cleared it out.